packages/react/src/components/Toolbar/ToolbarMenuItem.tsx

```diff
@@ -90,6 +90,13 @@ export interface ToolbarMenuItemProps
    */
   onClick?: ComponentEventHandler<ToolbarMenuItemProps>
 
+  /**
+   * Called when the menu inside the item opens or closes.
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props, with `menuOpen` reflecting the new state.
+   */
+  onMenuOpenChange?: ComponentEventHandler<ToolbarMenuItemProps>
```

Comment:
Shouldn't we invoke this somewhere?

Reply:
`onMenuOpenChange` vs `onMenuOpen` 😭

Reply:
What do you mean? Isn't it invoked on line 244?

Reply:
Ah got it, we are fixing typings, not adding new handler. Sorry for the coonfusion